eras/babbage/impl/cardano-ledger-babbage.cabal

```
@@ -73,7 +73,7 @@ library
         cardano-crypto-class,
         cardano-data >=1.2,
         cardano-ledger-allegra ^>=1.6,
-        cardano-ledger-alonzo >=1.9 && <1.11,
+        cardano-ledger-alonzo >=1.9 && <1.12,
```

`cardano-ledger-babbage` now requires these new changes to `EraPlutusTxInfo` type class, so we should reflect this with our bounds
```suggestion
        cardano-ledger-alonzo >=1.11 && <1.12,
```